frontend/src/metabase/query_builder/components/NewDatasetModal/NewDatasetModal.module.css

```diff
@@ -0,0 +1,8 @@
+.DatasetValueProp {
+  color: var(--mb-color-text-dark);
+  list-style-type: disc;
+  margin-left: 1rem;
+  padding-bottom: 1rem;
+  font-size: 14;
```

Comment:
Invalid value

Reply:
not really invalid, but 14pt indeed, nice catch!

Reply:
fixed